packages/contracts/contracts/UnirepApp.sol

```diff
@@ -26,9 +26,10 @@ contract UnirepApp {
     EpochKeyVerifierHelper internal epkHelper;
     EpochKeyLiteVerifierHelper internal epkLiteHelper;
 
+    uint256 private _postId;
```

Comment:
Agree on making it a public variable and change the name to latestPostId for clarity.

Reply:
Appreciate your advise. Just modified.